attelo/decoding/control.py

```diff
@@ -60,10 +60,14 @@ def _predict_relate(dpack, models):
     and an list of labels
     """
     if models.relate == 'oracle':
-        idxes = dpack.target
+        unrelated = dpack.label_number(UNRELATED)
         # pylint: disable=no-member
-        probs = np.ones(idxes.shape)
+        # treat unrelated as unknown (to avoid cutting links that may
+        # have been inserted by an non-oracle attachment label)
+        always_related = np.vectorize(lambda x: 0 if x == unrelated else x)
```

Comment:
I find the names quite unclear or misleading:
* `always_related`: `oracle_labeller` ?
* in the anonymous function, `x`: `lbl` ?
* `idxes`: `lbls`
* `probs`: `scores` (which is a general replacement to do)